packages/core/src/button-bar/button-bar.ts

```diff
@@ -19,6 +19,7 @@ export type ButtonBarAlign = 'start' | 'center' | 'end' | 'space-between';
  */
 export class ButtonBar extends LitElement {
   /** @private */
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Why do you need this? Does this error happen when there is no `.scss.js` file yet?

Reply:
I had a problem with committing merge main branch after solving conflicts, it was weird eslint error with `any` here.

Reply:
Yeah, I tried it myself: if you haven't run a `yarn build` yet, then there is no `*.scss.ts` file and it won't know the type of `styles`. I couldn't fix this other than just running the build.